opentelemetry-proto/src/proto.rs

```diff
@@ -1,65 +1,70 @@
 #[cfg(feature = "gen-tonic")]
 /// Generated files using [`tonic`](https://docs.rs/crate/grpcio) and [`prost`](https://docs.rs/crate/protobuf/latest)
 pub mod tonic {
+    macro_rules! include_tonic {
```

Comment:
IMO the `include_tonic!` macro doesn't carry its weight here. Instead, it would be better to put a `#[path = "proto/tonic/opentelemetry.proto.collector.logs.v1.rs"]` attribute on the `mod` declaration, which removes a layer of indirection and should make this more legible to Rust tooling (like IDEs).